qschmick/nj-test
# Fix/unit tests

## Description

Fixed failing unit test process and addressed bugs identified by tests.

## Types of changes

- [X] Chore (mundane code change)
- [ ] Maintenance (clean up)

## Checklist

- [X] Manually tested on locally or vhost
- [X] Added tests to code that was updated or edited

## Steps to test or review

1. Check out repo

1. Install composer deps

1. Run `./vendor/bin/phpunit` and see that all tests pass
